.changes/next-release/Feature-a4f4b3f0-0449-4e29-a8d1-f7d2c452ebc1.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Watch untitled yaml files for cloudformation/sam contents"
```

As a user, it is hard to see the meaning of this feature description. Suggestion:

```suggestion
	"description": "Enable syntax features for untitled (unsaved) cloudformation/sam yaml files"
```